### PR TITLE
Future postpones implemented.

### PR DESCRIPTION
**Pull request by AlexanderSuprunenko**

### CODE REVIEW

File: src/task.cpp [hunk 1]
```diff
@@ -174,11 +174,58 @@ void StateMachine::init_table()
 
 }
 
-TaskManager::TaskManager(const ConfigOpts& copts) : m_copts(copts)
+namespace detail
+{
+
+template<typename Uuid = Context::uuid_t, typename Clock = std::chrono::high_resolution_clock>
```
Comment (bitkis) [inline src/task.cpp]: use steady clock instead

File: src/task.cpp [hunk 2]
```diff
@@ -174,11 +174,58 @@ void StateMachine::init_table()
...
+class expiring_list
```
Comment (bitkis) [inline src/task.cpp]: naming convention is broken

File: src/task.cpp [hunk 3]
```diff
@@ -174,11 +174,58 @@ void StateMachine::init_table()
...
+    void add(const Uuid& uuid)
+    {
+        typename Clock::time_point now = Clock::now();
+        chop(now);
+        m_cont.emplace_back(std::make_pair(now + m_delta, uuid));
+    }
+    bool remove(const Uuid& uuid)
+    {
+        if(m_cont.empty()) return false;
+        chop();
```
Comment (bitkis) [inline src/task.cpp]: is it really needed??

**Comment by AlexanderSuprunenko** [comment]
Instead https://github.com/graft-project/graft-ng/pull/161

**Pull request closed without merging**